CIP-Improved_Rewards_Scheme_Parameters/README.md

```diff
@@ -0,0 +1,237 @@
+---
+CIP: 82
+Title: Improved Rewards Scheme Parameters
+Status: Project Area Not Enlisted for CIP Process
+Category: Reward-Sharing Scheme
+Authors:
+    - Tobias Fancee <tobiasfancee@gmail.com>
+Implementors:
+    - TBD
+Discussions:
+    - https://forum.cardano.org/t/cip-improved-rewards-scheme-parameters/112409
+Created: 2022-01-03
+License: CC-BY-4.0
+---
+
+# Improved Rewards Scheme Parameters
```

Comment:
```suggestion
```
Clarification from `CIP-0001` in meantime since you submitted your proposal precludes the H1 title (note in [CIP-0001 > Structure section](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#structure)) because this will generally be generated in software.